### ISSUE TITLE
Creating a project from renative-template-blank fails to run due to missing dependencies

### ISSUE THREAD

**Issue by dotted** [renative-org/renative#803]
**Describe the bug**
Using the `renative-template-blank` template when creating a new ReNative project will error out due to a missing dependency when running `rnv run` after creation using version 0.35.1

**To Reproduce**
1. Run `rnv new`
2. Use template `renative-template-blank` otherwise select the default suggest value by the tool
3. `cd` into newly create directory and run `rnv run`

**Expected behavior**
My browser opened showing the build output of my newly created project

**Screenshots or copy&paste**
```
➜  test rnv new

┌──────────────────────────────────────────────────────────────────────────────┐
│                                                                              │
│        ██████╗ ███████╗███╗   ██╗ █████╗ ████████╗██╗██╗   ██╗███████╗       │
│        ██╔══██╗██╔════╝████╗  ██║██╔══██╗╚══██╔══╝██║██║   ██║██╔════╝       │
│        ██████╔╝█████╗  ██╔██╗ ██║███████║   ██║   ██║██║   ██║█████╗         │
│        ██╔══██╗██╔══╝  ██║╚██╗██║██╔══██║   ██║   ██║╚██╗ ██╔╝██╔══╝         │
│        ██║  ██║███████╗██║ ╚████║██║  ██║   ██║   ██║ ╚████╔╝ ███████╗       │
│        ╚═╝  ╚═╝╚══════╝╚═╝  ╚═══╝╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═══╝  ╚══════╝       │
│                                                                              │
│        Version: 0.35.1                                                       │
│        https://renative.org                                                  │
│        🚀 Firing up!...                                                      │
│        $ rnv new                                                             │
│        Start Time: 2/10/2022, 1:45:32 AM                                     │
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘

[ task ] registerEngine:engine-core[1]
[ task ] configureRuntimeDefaults[1] appId:null
[ task ] checkAndMigrateProject[1]
[ task ] loadPluginTemplates[1]
[ task ] _parsePluginTemplateDependencies[1] scope:root
[ task ] parseRenativeConfigs[1]
[ task ] getWorkspaceDirPath[1]
[ task ] checkAndBootstrapIfRequired[1]
[ task ] loadEngines[1]
[ task ] findSuitableTask[1]
[ info ] Current Engine: engine-core path: undefined
[ task ] registerMissingPlatformEngines[1]
[ task ] initializeTask[1] new
[ task ] => [new]
[ task ] [new] taskRnvNew[1]
? What's your project Name? (no spaces, folder based on ID will be created in this directory) bhtest
? What's your project Title? Hello Renative
? What's your App ID? com.mycompany.bhtest
? What's your Version? 0.1.0
? What workspace to use? rnv
[ task ] [new] loadPluginTemplates[2]
[ task ] [new] _parsePluginTemplateDependencies[2] scope:root
[ task ] [new] parseRenativeConfigs[2]
[ task ] [new] getWorkspaceDirPath[2]
? What template to use? renative-template-blank
✔ Executing: npm view renative-template-blank versions
✔ Executing: npm dist-tag ls renative-template-blank
? What renative-template-blank version to use? 0.35.1 (HEAD: latest)
[ task ] [new] checkNpxIsInstalled[1]
✔ Executing: npx yarn add renative-template-blank@0.35.1
? What platforms would you like to use? android, androidtv, web, tizen, webos, windows, chromecast
? Do you want to set-up git in your new project? Yes
?
┌──────────────────────────────────────────────────────────────────────────────┐
│  🚀  ReNative Project Generator                                              │
│                                                                              │
├──────────────────────────────────────────────────────────────────────────────┤
│                                                                              │
│  Project Name (folder): bhtest                                               │
│  Workspace: rnv                                                              │
│  Project Title: Hello Renative                                               │
│  Project Version: 0.1.0                                                      │
│  App ID: com.mycompany.bhtest                                                │
│  Project Template: renative-template-blank@0.35.1                            │
│  Git Enabled: true                                                           │
│                                                                              │
│  Project Platforms:                                                          │
│  android, androidtv, web, tizen, webos, windows, chromecast                  │
│                                                                              │
│  Project Structure:                                                          │
│                                                                              │
│  bhtest                                                                      │
│   ├── appConfigs            # Application flavour configuration files/assets │
│   │   └── [APP_ID]          # Example application flavour                    │
│   │       ├── assets        # Platform assets injected to ./platformAssets   │
│   │       ├── builds        # Platform files injected to ./platformBuilds    │
│   │       ├── fonts             # Folder for all custom fonts                │
│   │       ├── plugins           # Multi-platform plugins injections          │
│   │       └── renative.json # Application flavour config                     │
│   ├── platformAssets        # Generated cross-platform assets                │
│   ├── platformBuilds        # Generated platform app projects                │
│   ├── src                   # Source code files                              │
│   ├── index.*.js            # Entry files                                    │
│   └── renative.json         # ReNative project configuration                 │
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘

Is all this correct? Yes
[ task ] [new] _generateProject:renative-template-blank:0.35.1[1]
[ task ] [new] checkAndCreateGitignore[1]
[ info ] [new] Your .gitignore is missing. CREATING...DONE
[ task ] [new] configureGit:.[1]
[ info ] [new] Your project does not have a git repo. Creating one...DONE
✔ Executing: git init
✔ Executing: git add -A
✔ Executing: git commit -m "Initial"
[ success ] [new] Your project is ready! navigate to project cd bhtest and run rnv run to see magic happen!
[ task ] <= new

 $ rnv new - Done! 🚀
┌──────────────────────────────────────────────────────────────────────────────┐
│  🚀  SUMMARY | 2/10/2022, 1:47:00 AM                                         │
│  $ rnv new                                                                   │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 0.35.1                                                    │
│  Engine: engine-core                                                         │
│  Build Scheme (-s): debug                                                    │
│  Env Info: linux | x64 | node v14.19.0                                       │
│  Executed Time: 0h:1m:27s:811ms                                              │
│                                                                              │
│ [ success ] [new] Your project is ready! navigate to project cd bhtest and run rnv run to see magic happen!
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘
➜  test cd bhtest
➜  bhtest git:(master) rnv run

┌──────────────────────────────────────────────────────────────────────────────┐
│                                                                              │
│        ██████╗ ███████╗███╗   ██╗ █████╗ ████████╗██╗██╗   ██╗███████╗       │
│        ██╔══██╗██╔════╝████╗  ██║██╔══██╗╚══██╔══╝██║██║   ██║██╔════╝       │
│        ██████╔╝█████╗  ██╔██╗ ██║███████║   ██║   ██║██║   ██║█████╗         │
│        ██╔══██╗██╔══╝  ██║╚██╗██║██╔══██║   ██║   ██║╚██╗ ██╔╝██╔══╝         │
│        ██║  ██║███████╗██║ ╚████║██║  ██║   ██║   ██║ ╚████╔╝ ███████╗       │
│        ╚═╝  ╚═╝╚══════╝╚═╝  ╚═══╝╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═══╝  ╚══════╝       │
│                                                                              │
│        Version: 0.35.1                                                       │
│        https://renative.org                                                  │
│        🚀 Firing up!...                                                      │
│        $ rnv run                                                             │
│        Start Time: 2/10/2022, 1:47:12 AM                                     │
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘

[ task ] registerEngine:engine-core[1]
[ task ] configureRuntimeDefaults[1] appId:null
[ task ] checkAndMigrateProject[1]
[ task ] loadPluginTemplates[1]
[ task ] _parsePluginTemplateDependencies[1] scope:root
[ task ] parseRenativeConfigs[1]
[ task ] getWorkspaceDirPath[1]
[ task ] _generatePlatformTemplatePaths[1]
[ task ] checkAndBootstrapIfRequired[1]
┌──────────────────────────────────────────────────────────────────────────────┐
│  🚀  SUMMARY | 2/10/2022, 1:47:12 AM                                         │
│  $ rnv run                                                                   │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 0.35.1                                                    │
│  Project Name ($package.name): undefined                                     │
│  Project Version ($package.version): undefined                               │
│  Workspace ($.workspaceID): rnv                                              │
│  Build Scheme (-s): debug                                                    │
│  Env Info: linux | x64 | node v14.19.0                                       │
│  Executed Time: 0h:0m:0s:36ms                                                │
│                                                                              │
│ [ error ] Cannot find module '@rnv/integration-vercel'
Require stack:
- /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/dist/core/integrationManager/index.js
- /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/dist/cli/index.js
- /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/dist/index.js
- /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/bin/index.js
Error: Cannot find module '@rnv/integration-vercel'
Require stack:
- /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/dist/core/integrationManager/index.js
- /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/dist/cli/index.js
- /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/dist/index.js
- /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/bin/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at /home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/dist/core/integrationManager/index.js:13:44
    at Array.forEach (<anonymous>)
    at _callee$ (/home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/dist/core/integrationManager/index.js:11:41)
    at tryCatch (/home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/node_modules/regenerator-runtime/runtime.js:63:40)
    at Generator.invoke [as _invoke] (/home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/node_modules/regenerator-runtime/runtime.js:294:22)
    at Generator.next (/home/dotted/.nvm/versions/node/v14.19.0/lib/node_modules/rnv/node_modules/regenerator-runtime/runtime.js:119:21)
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘
➜  bhtest git:(master)
```

**Desktop (please complete the following information):**

-   OS: Ubuntu 20.04 running on WSL2
-   Node Version 14.19
-   RNV Version 0.35.1

**Additional context**
The `renative-template-hello-world` seems to work fine though.

**Comment by kasinskas** [comment]
Hey @dotted, should be fixed in 0.35.2

**Comment by dotted** [comment]
Confirmed fixed, thanks!
